src/wp-includes/script-loader.php

```diff
@@ -1292,7 +1292,7 @@ function wp_default_scripts( $scripts ) {
 			'close'                   => __( 'Close' ),
 			'action'                  => __( 'Action' ),
 			'discardChanges'          => __( 'Discard changes' ),
-			'cheatin'                 => __( 'Something went wrong.' ),
+			'cheatin'                 => __( 'An unexpected error occurred. Please check your permissions and try again.' ),
```

Comment:
Permissions are not the only situation that can trigger 'cheatin', so that should probably not be implied. It's mostly about invalid nonce values.